### ISSUE TITLE
Autogenerate author for new taxa

### ISSUE THREAD

**Issue by sharpphyl** [ArctosDB/arctos#1250]
As I remember, Arctos used to autogenerate the Author field when a new taxon was added.  The field was filled with the Author name from the source we cloned.  Perhaps it was when we stated to autogenerate the Display Name that the Author field was no longer populated.

Having the author is very important to know that you have the correct taxon for a species.  I am manually adding that to all the taxa I create, but it would be a huge convenience (and eliminate typos and other possible errors) if this field could be populated the same way each field in the classification is.

**Comment by dustymc** [comment]
Display name is built from author(s), it does not replace them.

While trying to bring data into the hierarchical tool and deal with #891, I noticed that we were being incredibly inconsistent in local edits, and of course a hierarchy cannot deal with inconsistent data. That lead to tightening up what's accepted in local classifications, which I believe lead to the single-edit form just ignoring unfamiliar (not in https://arctos.database.museum/info/ctDocumentation.cfm?table=CTTAXON_TERM) term types.

I do remember "authors" from that experience - there were (probably still are) many where the author-bearing namestring from GN was moved into "author_text" leading to display_name being generated as "Some taxa Some taxa author."

I can try to guess at some stuff or play with the single-record form or whatever, but that'll still lead to inconsistent data and prevent users from finding our specimens (with our data anyway!). I'd rather just develop whatever it takes to move all edits to the hierarchical editor and deprecate editing single taxa.
